src/timeline/media/css/main.css

```diff
@@ -303,7 +303,11 @@ img {
   margin-right: 1px;
 }
 
-.razor_cursor {
+.webkit .razor_cursor {
+  cursor: url(../images/razor_line_with_razor.png), default !important;
```

Comment:
Apparently webkit does not support SVG files for CSS cursors. I tried a bunch of different test files, different sizes, and never got one to appear. So using a PNG instead (but only for webkit).

Reply:
Bizarre! But good to know!